youtube_dl/extractor/cspan.py

```diff
@@ -9,42 +9,46 @@
     find_xpath_attr,
     smuggle_url,
     determine_ext,
+    ExtractorError,
 )
 from .senateisvp import SenateISVPIE
 
 
+def get_text_attr(d, attr):
```

Comment:
For utility functions, either put it in an ```InfoExtractor``` subclass as ```staticmethod``` or ```classmethod``` if it's a site-specific one, or put it in [```youtube_dl/utils.py```](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/utils.py) if it's a generic one.